Services/AssessmentQuestion/examples/class.exObjQuestionPoolGUI.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+/**
+ * When a component consumes the assessment question service for purposes
+ * of authoring and managing questions like the current question pool object,
+ * it is neccessary to handle the following use cases.
+ */
+class exObjQuestionPoolGUI
+{
+	/**
+	 * The question creation and editing is handled by the the ilAsqQuestionAuthoring interface.
+	 * It is relevant for the request flow using the ILIAS control structure. This interface
+	 * will be implemented from the current question GUI classes in the future and provides
+	 * the executeCommand method that internally performs the different commands.
+	 *
+	 * To integrate the workflow of authoring questions to a consumer, the consumer's GUI class
+	 * must implement a suitable forward case in its own executeCommand method.
+	 *
+	 * Since the consumer MUST not know about any concrete question type's ilAsqQuestionAuthoring implementation
+	 * a dynamic switch-case expression is neccessary. The ilAsqService class provides such an expression
+	 * in form of a static method. It returns the lowercase class name for the corresponding
+	 * question type authoring class or the interface name when the current next class in the control flow
+	 * does not relate to any question type authoring class.
+	 */
+	public function executeCommand()
+	{
+		global $DIC; /* @var ILIAS\DI\Container $DIC */
```

Comment:
This placement of the php-doc somehow looks funny to me, why not place it above the variable?